src/requirements/reqs-functions.ts

```diff
@@ -207,22 +117,20 @@ export function computeRequirements(
   // prepare grouped fulfillment summary
   const groups: GroupedRequirementFulfillmentReport[] = [];
 
-  // PART 1: check university requirements
-  groups.push({
-    groupName: 'University',
-    specific: null,
-    reqs: computeUniversityRequirementFulfillments(coursesTaken)
-  });
-
-  // PART 2: check college & major & minor requirements
```

Comment:
Can you relabel the parts now that you deleted part 1?

Reply:
It's relabeled to "check university & college & major & minor requirements" but should i add a comment saying it's updated or ?

Reply:
Oops sorry I meant change the numbers! Since we got rid of part 1, this is now part 1 right?